src/viam/ur/module/ur_arm.cpp

```diff
@@ -84,7 +91,7 @@ T find_config_attribute(const ResourceConfig& cfg, const std::string& attribute)
     return *val;
 }
 
-enum class TrajectoryStatus { k_running = 1, k_cancelled = 2, k_stopped = 3 };
+enum class TrajectoryStatus : std::uint8_t { k_running = 1, k_cancelled = 2, k_stopped = 3 };
```

Comment:
I actually disagree with the complaint here. We don't care about the size of this. Let the compiler pick. Can you NOLINT it? I might care in headers so I don't want to disable the check completely.